components/actions_menu/actions_menu.test.tsx

```diff
@@ -67,24 +67,47 @@ describe('components/actions_menu/ActionsMenu', () => {
         expect(wrapper.find('#divider_post_post_id_1_marketplace').exists()).toBe(true);
     });
 
-    test('no actions - menu should be visible to sysadmin', () => {
+    test('has actions - sysadmin - should show actions and app marketplace', () => {
         const wrapper = shallowWithIntl(
             <ActionsMenu {...baseProps}/>,
         );
-        expect(wrapper.find('#marketPlaceButton').exists()).toBe(true);
+        wrapper.setProps({
+            pluginMenuItems: dropdownComponents,
+        });
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('has actions - end user - should not show actions and app marketplace', () => {
+        const wrapper = shallowWithIntl(
+            <ActionsMenu {...baseProps}/>,
+        );
+        wrapper.setProps({
+            pluginMenuItems: dropdownComponents,
+            isSysAdmin: false,
+        });
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('no actions - sysadmin - menu should show visit marketplace', () => {
+        const wrapper = shallowWithIntl(
+            <ActionsMenu {...baseProps}/>,
+        );
+        expect(wrapper).toMatchSnapshot();
     });
 
-    test('no actions - menu should not be visible to end user', () => {
+    test('no actions - end user - menu should not be visible to end user', () => {
         const wrapper = shallowWithIntl(
             <ActionsMenu {...baseProps}/>,
         );
         wrapper.setProps({
             isSysAdmin: false,
         });
-        expect(wrapper.find('#marketPlaceButton').exists()).toBe(false);
+
+        // menu should be empty
+        expect(wrapper.debug()).toMatchSnapshot();
```

Comment:
Just for my knowledge: why debug here?

Reply:
`.debug()` will print the contents of the rendered component, which is empty in this case.